packages/realm/RealmJS.podspec

```diff
@@ -50,21 +57,43 @@ Pod::Spec.new do |s|
 
   s.pod_target_xcconfig    = {
                                 # Setting up clang
-                                'CLANG_CXX_LANGUAGE_STANDARD' => 'c++17',
+                                'CLANG_CXX_LANGUAGE_STANDARD' => 'c++20',
                                 'CLANG_CXX_LIBRARY' => 'libc++',
                                 # Setting the current project version and versioning system to get a symbol for analytics
                                 'CURRENT_PROJECT_VERSION' => s.version,
+                                'REALM_BUILD_CORE' => build_realm_core,
+                                'CMAKE_PATH' => cmake_path,
                                 'VERSIONING_SYSTEM' => 'apple-generic',
+                                'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) REALM_ENABLE_SYNC=1',
+                                'GCC_SYMBOLS_PRIVATE_EXTERN' => 'YES',
                                 # Header search paths are prefixes to the path specified in #include macros
                                 'HEADER_SEARCH_PATHS' => [
                                   '"$(PODS_TARGET_SRCROOT)/react-native/ios/RealmReact/"',
-                                  '"$(PODS_ROOT)/Headers/Public/React-Core/"'
-                                  #"'#{app_path}/ios/Pods/Headers/Public/React-Core'" # Use this line instead of 👆 while linting
+                                  '"$(PODS_TARGET_SRCROOT)/react-native/ios/build/include/"',
+                                  '"$(PODS_TARGET_SRCROOT)/binding/"',
+                                  '"$(PODS_TARGET_SRCROOT)/bindgen/src/"',
+                                  '"$(PODS_TARGET_SRCROOT)/bindgen/vendor/realm-core/bindgen/src/"'
                                 ].join(' ')
                               }
+  # Create placeholders for vendored_libraries, so they are added to the xcode project
+  s.prepare_command = <<-EOS
+  source "#{__dir__}/scripts/generate-dummy-libs.sh"
+  source "#{__dir__}/scripts/generate-input-list.sh"
+  EOS
 
-  # TODO: Consider providing an option to build with the -dbg binaries instead
-  s.vendored_frameworks = 'react-native/ios/realm-js-ios.xcframework'
+  s.vendored_libraries = "react-native/ios/lib/*.a"
 
   s.dependency 'React'
+
+
+  #Post install script
+  s.script_phase = {
+    :name => 'Retrieve libraries and headers',
+    :execution_position => :before_compile,
+    :input_file_lists => ["$(PODS_TARGET_SRCROOT)/react-native/ios/input-files.xcfilelist"],
+    :output_file_lists => ["$(PODS_TARGET_SRCROOT)/react-native/ios/output-files.xcfilelist"],
+    :script => <<-EOS
+    source "$PODS_TARGET_SRCROOT/scripts/generate-ios-libs.sh"
```

Comment:
I don't think "generate" is the best name for a script that potentially downloads 🤔

Reply:
it downloads or builds.  What verb should we use for that?

Reply:
`install`?

Reply:
Fetch, Build, Assemble, Construct, Retrieve, Synthesize, Compile, Download, Install, Resolve, Sync, Fabricate, Configure, Provision

Reply:
@realm/realm-js-team thoughts?

Reply:
@realm/realm-js-team thoughts?

Reply:
How about `download-or-build-ios.sh`?

Reply:
Sure